mandatory/1-fix-functions.js

```diff
@@ -51,80 +39,45 @@ function get3rdIndex(arr) {
 
 /* ======= TESTS - DO NOT MODIFY ===== */
 
-const util = require("util");
-
-function test(test_name, actual, expected) {
-  let status;
-
-  let isEqual;
-  if (Array.isArray(expected)) {
-    isEqual = arraysEqual(actual, expected);
-  } else {
-    isEqual = actual === expected;
-  }
-
-  if (isEqual) {
-    status = "PASSED";
-  } else {
-    status = `FAILED: expected: ${util.inspect(
-      expected
-    )} but your function returned: ${util.inspect(actual)}`;
-  }
-
-  console.log(`${test_name}: ${status}`);
-}
-
-function arraysEqual(a, b) {
-  if (a === b) return true;
-  if (a == null || b == null) return false;
-  if (a.length != b.length) return false;
-
-  for (let i = 0; i < a.length; ++i) {
-    if (a[i] !== b[i]) return false;
-  }
-
-  return true;
-}
-
-test("mood function works for true", mood(true), "I am happy");
-test("mood function works for false", mood(false), "I am not happy");
-test(
-  "greaterThanTen function works for 11",
-  greaterThan10(11),
-  "num is greater than 10"
-);
-test(
-  "greaterThanTen function works for 10",
-  greaterThan10(10),
-  "num is not big enough"
-);
-test(
-  "greaterThanTen function works for 9",
-  greaterThan10(9),
-  "num is not big enough"
-);
-test("sortArray function works", sortArray(["a", "n", "c", "e", "z", "f"]), [
-  "a",
-  "c",
-  "e",
-  "f",
-  "n",
-  "z",
-]);
-
-let numbers = [1, 2, 3, 4, 5, 6, 7, 8];
-test("first5 function works", first5(numbers), [1, 2, 3, 4, 5]);
-if (!arraysEqual(numbers, [1, 2, 3, 4, 5, 6, 7, 8])) {
-  console.log("PROBLEM: first5 changed its input array - it shouldn't!");
-}
-
-test(
-  "get3rdIndex function works - case 1",
-  get3rdIndex(["fruit", "banana", "apple", "strawberry", "raspberry"]),
-  "strawberry"
-);
-test(
-  "get3rdIndex function works - case 2",
-  get3rdIndex([11, 37, 62, 18, 19, 3, 30]),
-  18
-);
+const {expect, test} = require("@jest/globals");
+
+test("mood function works for true", () => {
+    expect(mood(true)).toEqual("I am happy");
+});
+
+test("mood function works for false", () => {
+    expect(mood(false)).toEqual("I am not happy");
+});
+
+test("greaterThanTen function works for value greater than 10", () => {
+    expect(greaterThan10(11)).toEqual("num is greater than 10");
```

Comment:
I added an extra one - tbh though I think the "fix basic syntax" and "fix functions" tests really are better judged by human eyes than tests - there's very specific _exact source code_ we're expecting the trainees to write here, rather than a more vague specification we're giving them for the function...